mindsdb/integrations/handlers/notion_handler/notion_handler.py

```diff
@@ -0,0 +1,231 @@
+import time
+import pandas as pd
+from collections import defaultdict
+
+from pydantic import BaseModel, Extra
+from notion_client import Client
+
+from mindsdb.utilities import log
+from mindsdb.integrations.libs.api_handler import APIHandler
+from mindsdb.integrations.libs.response import HandlerStatusResponse as StatusResponse
+
+from .notion_table import (
+    NotionBlocksTable,
+    NotionCommentsTable,
+    NotionDatabaseTable,
+    NotionPagesTable,
+)
+
+
+class NotionHandlerArgs(BaseModel):
+    target: str = None
+    api_token: str = None
+
+    class Config:
+        # for all args that are not expected, raise an error
+        extra = Extra.forbid
+
+
+class NotionHandler(APIHandler):
```

Comment:
The `native_query()` method is missing from the handler. This is required, please implement it.

Reply:
thanks, I have added it. But I have added that we can query directly Python-like queries from SQL is that fine?
example:

```sql
SELECT * FROM notion_test 
(
    databases.query(database_id=b144844d418f404a8f0f1da2a63cdf7c)
);
SELECT * FROM notion_test 
(
    comments.retrieve(block_id=6d1480e0bf4b46e1a71be093f105d654)
);
```